README.md

```diff
@@ -4,42 +4,52 @@ This is a skeleton for confidential Oasis dApps in Python.
 
 ## Prerequisites
 
-This project was tested on python 3.10, but should work with most python3 versions. 
+This project was tested on python 3.10, but should work with most 
+python3 versions. 
 Use pyenv to handle multiple python installations.
 
-1. Initialize an environment using preferred environment manager (venv, pipx...) ```python3 -m venv my_env``` 
-and install the python sapphire wrapper. 
-The library is currently **NOT** included in the PyPI repository so make sure to only install the local .whl version.
+## Installation
 
-2. Make sure to follow the [instructions](https://github.com/oasisprotocol/sapphire-paratime/tree/main/clients/py "Py client wrapper") and build the .whl file correctly. 
-In the Py client wrapper dist/ folder, find the .whl file and install 
-it by running ```pip install <path_to_whl_file>```.
+1. Initialize an environment using preferred environment manager 
+(venv, pipx...) ```python3 -m venv my_env```. 
+There are two ways to install the wrapper client:
 
-3. Install the packages in requirements.txt ```pip install -r requirements.txt```
+    - Build the .whl file from the source code in the 
```

Comment:
Let's just put the `oasis-sapphire-py` into `requirements.txt`. Installation from wheel should be put into the py client's README.

Reply:
Updated README installation guide.